specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -0,0 +1,6389 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Hostpool update full properties including session host config, hostpool update parameters, and migration progress.",
+            "schema": {
+              "$ref": "#/definitions/UpdateStatus"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/updateValidationResults/{updateValidationInstanceId}": {
+      "get": {
+        "tags": [
+          "HostPoolUpdate"
+        ],
+        "operationId": "UpdateValidationResults_Get",
+        "description": "Operation status of a validate hostpool update.",
+        "x-ms-examples": {
+          "UpdateValidationResults_Get": {
+            "$ref": "./examples/UpdateValidationResults_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HostPoolUpdateValidationInstanceIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Errors and warnings returned by hostpool update validation.",
+            "schema": {
+              "$ref": "#/definitions/HostPoolUpdateValidationResponse"
+            }
+          },
+          "202": {
+            "description": "Async operation in progress."
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/updateResults/{updateInstanceId}": {
```

Comment:
Endpoint duplication? this one and the one above it? @datr93

Reply:
on for validation and one for actual update. we need one for both according to arm team.

Reply:
validate responds with  HostPoolUpdateValidationResponse initiate responds with HostPoolUpdateFullProperties